README.md

```diff
@@ -138,11 +138,18 @@ cd $GOPATH/src/github.com/kubeedge/kubeedge
 
 #### Run as a binary
 
++ Build Cloud and edge
+
+    ```shell
+    cd $GOPATH/src/github.com/kubeedge/kubeedge
+     make or `make all `
```

Comment:
I think provide an option is enough here. So please either choose `make` or `make all`.

Reply:
ok will chose make